adapters/gamma/gamma.go

```diff
@@ -149,7 +149,7 @@ func (a *GammaAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapt
 		errs = append(errs, err)
 		return nil, errs
 	}
-	var invalidImpIndex = make([]int, 0, 0)
+	var invalidImpIndex = make([]int, 0)
```

Comment:
Let's use `var invalidImpIndex []int` instead to avoid the allocation when there are no invalid imps.

Reply:
done in 25808d59